tools/Jupyter/kernel/clingkernel.py

```diff
@@ -123,7 +124,15 @@ def __init__(self, **kwargs):
                 self.libclingJupyter = ctypes.CDLL(clingInstDir + "/lib/libclingJupyter." + ext,
                                                    mode = ctypes.RTLD_GLOBAL)
                 break
-
+                
+        for ext in ['so', 'dylib', 'dll']:
+            libFilename = clingInstDir + "/libexec/lib/libclingJupyter." + ext
```

Comment:
Good idea. I made the requested change, with a single modification. I am looping over the full path instead of `'lib'` and `'libexec'`. This will allow others to add paths in the future. If it is located in multiple paths, it will find the first one in the list.